packages/transaction-controller/src/TransactionController.ts

```diff
@@ -124,6 +124,18 @@ export interface TransactionState extends BaseState {
   methodData: { [key: string]: MethodData };
 }
 
+/**
+ * @type AddTransactionOptions
+ *
+ * Transaction controller addTransaction method options
+ * @param origin - The domain origin to append to the generated TransactionMeta.
+ * @param deviceConfirmedOn - An enum to indicate what device the transaction was confirmed to append to the generated TransactionMeta.
+ */
+type AddTransactionOptions = {
+  origin?: string;
```

Comment:
Minor, but as this list will likely grow, can we keep them in alphabetical order?

Reply:
Done